mmcv/ops/csrc/roi_align_rotated_cuda_kernel.cuh

```diff
@@ -0,0 +1,199 @@
+#ifndef ROI_ALIGN_ROTATED_CUDA_KERNEL_CUH
```

Comment:
Should this file recognize https://github.com/facebookresearch/detectron2/blob/master/detectron2/layers/csrc/ROIAlignRotated/ROIAlignRotated_cpu.cpp?